docs/docs/en/guide/howto/datasource-setting.md

```diff
@@ -92,9 +92,9 @@ For PostgreSQL:
 # for postgresql
 export DATABASE=${DATABASE:-postgresql}
 export SPRING_PROFILES_ACTIVE=${DATABASE}
-export SPRING_DATASOURCE_URL="jdbc:postgresql://127.0.0.1:5432/dolphinscheduler"
+export SPRING_DATASOURCE_URL="jdbc:postgresql://127.0.0.1:5432/dolphinscheduler?stringtype=unspecified"
```

Comment:
why do we need to add this in PG? Does there exist any SQL use string as int/long/xx ?

Reply:
Maybe insert to a timestamp used string. When I run a data quality task without that, I got an error: `column "data_time" is of type timestamp without time zone but expression is of type character varying`.

Reply:
MySQL doesn't have those problems so I'm using MySQL now. Honestly I think it's better to use MySQL here.

Reply:
Is this caused by using the incorrect data type in SQL?  It's better to fix the SQL rather than hide that.